src/components/landing-page/landing-page.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import './landing-page.scss';
+
+class LandingPage extends Component {
```

Comment:
As we discussed it might be better to turn this into a presentational component / function instead of a class component as we are not using any state here